server.js

```diff
@@ -17,4 +20,9 @@ app.all("/*", (req, res, next) => {
 app.use(handlePsqlErrors);
 app.use(handleCustomErrors);
 
+//listening
```

Comment:
If we aren't using this then I would get rid of it. We are using supertest to test client requests, so a listen.js isn't really needed as the PORT connection will be set up automatically by supertest. If you are using insomnia then having this set up would make sense, but it isn't necessary!